test/functionalTest/cases/1005_GET_v2_entities_string/1005_GET_v2_entities_string.test

```diff
@@ -0,0 +1,137 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entities V2 string
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v1/updateContext
+# 03. GET /v2/entities
+#
+
+echo "01. POST /v1/updateContext"
+echo "=========================="
+payload='{
+  "contextElements": [
+    {
+      "id":   "8787GHY",
+      "type": "Car",
+      "attributes" : [
+        {
+          "name" : "speed",
+          "type": "number",
+          "value" : 150
+        },
+        {
+          "name" : "brand",
+          "type": "string",
```

Comment:
Maybe some of the attributes should use an "implicit string" type, in order to check that it is also correctly rendered. E.g: `"type": "myString"`.